src/main/java/andersen/AndersenTasks/collections/CustomHashSet.java

```diff
@@ -0,0 +1,128 @@
+package andersen.AndersenTasks.collections;
+
+
+import java.util.*;
```

Comment:
Use full qualified imports instead of generic one with asterisk for better readability